redis/client.py

```diff
@@ -283,7 +283,13 @@ def nativestr_or_none(response):
 def parse_stream_list(response):
     if response is None:
         return None
-    return [(r[0], pairs_to_dict(r[1])) for r in response]
+    data = []
+    for r in response:
+        if r is not None:
+            data.append((r[0], pairs_to_dict(r[1])))
+        else:
+            data.append(None)
```

Comment:
I think this should be: `data.append((None, None))`

This would allow users to iterate over XCLAIM results like:

```python
for message_id, message in r.xclaim(...):
    ...
```

Reply:
Alright. Makes sense to me too. Thank you